pkg/controller/add_cluster.go

```diff
@@ -81,29 +80,6 @@ func (c *Controller) EnsurePlacement(cluster *myspec.M3DBCluster) error {
 	return nil
 }
 
-// EnsureNamespace will retrieve current namespaces to ensure one matches the
-// cluster name or create a new namespace to match the cluster name. Returns a
-// bool indicating whether a namespace was created and any errors encountered.
-func (c *Controller) EnsureNamespace(cluster *myspec.M3DBCluster) (bool, error) {
```

Comment:
So just to get my understanding correct, we're removing this function because the namespace is already "ensured" in the yaml with its own customizations?

Reply:
Ah yeah, I removed it because I thought that 1) now that we have multiple namespaces just `EnsureNamespace` didn't fit that description, and 2) since we're more than just ensuring it's there we might end up removing some, so "reconcile" felt more fitting. We still have to take what's in the YAML and "ensure" they end up in the cluster, but it's a bit more of a dynamic process